source/components-and-apis.rst

```diff
@@ -355,12 +341,11 @@ dojot.
 .. _Messages - iotagent-mosca: http://dojotdocs.readthedocs.io/projects/iotagent-mosca/en/latest/operation.html#sending-messages-to-other-components-via-kafka
 .. _GitHub - iotagent-mosca: https://github.com/dojot/iotagent-mosca
 
-.. _GitHub - EJBCA-REST: https://github.com/dojot/ejbca-rest
-.. _API - EJBCA-REST: https://dojot.github.io/ejbca-rest/apiary_v0.4.3.html
-
 .. _GitHub - Data Manager: https://github.com/dojot/data-manager
 .. _API - Data Manager: https://dojot.github.io/data-manager/apiary_v0.4.3.html
 
 .. _GitHub - Cron: https://github.com/dojot/cron
 .. _API - Cron: https://dojot.github.io/cron/apiary_v0.4.3.html
 
+.. _GitHub - x509-identity-mgmt: https://github.com/dojot/dojot/tree/development/x509-identity-mgmt
+.. _API - x509-identity-mgmt: https://dojot.github.io/dojot/x509-identity-mgmt/apiary_release-v0.5.0-alpha.html
```

Comment:
The final link should be https://dojot.github.io/dojot/x509-identity-mgmt/apiary_v0.5.0.html